osmaxx/excerptexport/views.py

```diff
@@ -125,13 +126,13 @@ def get_context_data(self, **kwargs):
 
     def get_queryset(self):
         return sorted(
-            super().get_queryset().filter(pk__in=self.excerpt_ids), key=lambda x: self.excerpt_ids.index(x.pk)
+            super().get_queryset().filter(pk__in=self.excerpt_ids).defer('bounding_geometry'), key=lambda x: self.excerpt_ids.index(x.pk)
         )
 
     def _get_exports_for_excerpt(self, excerpt):
         return self.get_user_exports().\
             filter(extraction_order__excerpt=excerpt).\
-            select_related('extraction_order', 'extraction_order__excerpt', 'output_file')
+            select_related('extraction_order', 'extraction_order__excerpt', 'output_file').defer('extraction_order__excerpt__bounding_geometry')
```

Comment:
Cool, didn't know `defer` can work on related fields and influence `select_related`.